.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+Before submitting a pull request, please make sure the following has been completed:
+
+- You've rebased or merged in the latest version of master.
+- You've added or updated tests around this change.
+- You've run `npm run check` locally and it's passed.
+- You've run `npm run jsdoc` to update the README.md.
+- Explain in detail what this pull request accomplishes and why it's necessary.
+- If you're fixing an issue include fixes #1234 in this text box.
```

Comment:
`- If you're fixing an issue, include "Fixes #1234" in this text box.`

Reply:
updated